### PR TITLE
[Enhancement]  Migrated bottom label to card drawer

### PR DESCRIPTION
**Pull request by daniivillegas**
Se integra BottomLabel de CardDrawer

High credit card with combo 

High debit card with combo

High credit card v2

Medium credit card v2

Low credit card v2

High debit card v2

Medium debit card v2

Low debit card v2

High account money v2

Medium account money v2

Low account money v2

Pix

Medium credit card disabled

High credit card disabled

Low credit card disabled

Medium debit card disabled

High debit card disabled

Low debit card disabled

Idc high

Idc low

### CODE REVIEW

File: px-checkout/src/main/res/layout/px_consumer_credits_card_view.xml [hunk 1]
```diff
@@ -73,7 +73,5 @@
             app:layout_constraintVertical_bias="1.0"
             tools:text="Al confirmar, aceptás los términos generales y las condiciones particulares de este préstamo." />
 
-        <include layout="@layout/px_highlight_text" />
```
Comment (jorGonzalez291292) [inline px-checkout/src/main/res/layout/px_consumer_credits_card_view.xml:76]: Habíamos dicho que no se usaba el label para consumer credits, no?

File: px-checkout/build.gradle [hunk 2]
```diff
@@ -50,7 +50,7 @@ dependencies {
 
     implementation "com.google.code.gson:gson:$gson"
     implementation "com.mercadolibre.android:ui:$ui"
-    implementation "com.mercadolibre.android:carddrawer:$cardDrawer"
+    implementation ("com.mercadolibre.android:carddrawer:$cardDrawer")
```
[the remark this answers is not in the document]
Comment (daniivillegas) [reply px-checkout/build.gradle:53]: colgue en borrarlo cuando hice el force amigacho, ahora lo borro

File: px-checkout/src/main/res/layout/px_consumer_credits_card_view.xml (hunk 1 above)
Comment (daniivillegas) [reply px-checkout/src/main/res/layout/px_consumer_credits_card_view.xml:76]: nop

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/PaymentMethodFragment.java [hunk 3]
```diff
@@ -8,29 +8,29 @@
 import android.view.View;
 import android.view.accessibility.AccessibilityEvent;
 import android.widget.ImageView;
+
```
Comment (francofusaro) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/PaymentMethodFragment.java:11]: ?

Comment (daniivillegas) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/slider/PaymentMethodFragment.java:11]: gracias por la info :P

**Review by jorGonzalez291292 — APPROVED** [review]

**Pull request merged**